templates/cloud/openstack/index.html

```diff
@@ -26,13 +26,14 @@
             </ul>
             <p><a href="/download/cloud" class="button--primary">Get started with OpenStack Autopilot</a></p>
             <p><a href="/cloud/openstack/autopilot">Learn more about Autopilot&nbsp;&rsaquo;</a></p>
-    
+
         </div>
         <div class="four-col last-col">
             {% comment %} might want to add this image instead of the chart - {{ ASSET_SERVER_URL }}845e046c-partner-openstack.png
             {% endcomment %}
             <figure id="ubuntu-pie" class="ubuntu-pie"></figure>
-            <p class="note">* <a href="http://www.openstack.org/assets/survey/Public-User-Survey-Report.pdf" class="external">Among OpenStack deployments with more than 1,000 users, OpenStack survey &mdash; October 2015 [PDF 6.7MB]</a></p>
+            <p class="note">* Among OpenStack deployments with more than 1,600 users,
```

Comment:
There's an asterisk here, as if it's referencing some other asterisk on that page, but there doesn't seem to be one. I've discussed with Peter he agreed the `*` should be removed.